comment out audio phase meter

dericed opened the pull request:
filter was changed from A->V to A->N

dericed commented:
address https://github.com/bavc/qctools/issues/314

richardpl commented:
Why was this merged? This is nonsense. Filter should work still. You just need to add option to enable video output.

dericed commented:
IIRC when video is enabled the filter also outputs audio. afk

richardpl commented:
Then use anullsink. Same applies to ebur128 video output.

dericed commented:
I found that I can get the video output back if I label the output as `[out1]`, such as:
`ffplay -f lavfi -i aevalsrc="sin(420*2*PI*t)|cos(430*2*PI*t)",aphasemeter=video=1[out1]`

Could you suggest how to add in anullsink in this sort of example.

dericed commented:
Not the most efficient, but I got this working:
`ffplay -f lavfi -i aevalsrc="sin(420*2*PI*t)|cos(430*2*PI*t),aphasemeter=video=1[out0][out1];[out0]anullsink;[out1]copy"`

dericed commented:
I fixed aphasemeter using the approach referenced above. Then I rebased, squashed over this pull request's commit, and pushed with force. So now there's https://github.com/bavc/qctools/commit/2f25c8b09144ac54c3b3d52a1476b279cc1c957e.
